tests/validation/tests/v3_api/test_create_ha.py

```diff
@@ -246,33 +248,34 @@ def install_eks_ingress():
                             DATA_SUBDIR + "/eks_nlb.yml")
 
 
-def set_route53_with_eks_ingress():
-    kubectl_ingress = "kubectl get ingress -n cattle-system -o " \
-                      "jsonpath=\"" \
-                      "{.items[0].status.loadBalancer.ingress[0].hostname}\""
-    ingress_address = run_command_with_stderr(export_cmd
-                                              + " && " +
-                                              kubectl_ingress).decode()
-    AmazonWebServices().upsert_route_53_record_cname(RANCHER_HA_HOSTNAME,
-                                                     ingress_address)
-    time.sleep(60)
-
-
-def set_route53_with_aks_ingress():
-    kubectl_ingress = "kubectl get svc -n ingress-nginx " \
-                      "ingress-nginx-controller -o " \
-                      "jsonpath=\"" \
-                      "{.status.loadBalancer.ingress[0].ip}\""
-    time.sleep(10)
-    ingress_address = run_command_with_stderr(export_cmd
-                                              + " && " +
-                                              kubectl_ingress).decode()
-
-    print("AKS INGRESS ADDRESS:")
+def set_route53_with_ingress():
+    ingress_address = None
+    if RANCHER_LOCAL_CLUSTER_TYPE == "EKS":
+        kubectl_ingress = "kubectl get ingress -n cattle-system -o " \
+                          "jsonpath=\"" \
+                          "{.items[0].status.loadBalancer.ingress[0].hostname}\""
+        time.sleep(10)
+        ingress_address = run_command_with_stderr(export_cmd
+                                                  + " && " +
+                                                  kubectl_ingress).decode()
+        AmazonWebServices().upsert_route_53_record_cname(RANCHER_HA_HOSTNAME,
+                                                         ingress_address)
+
```

Comment:
Nit: else if for these comparisons, and maybe a graceful failure if `RANCHER_LOCAL_CLUSTER_TYPE` is an unexpected value

Reply:
I just refactored the code to use `elif` it was called out before the providers code was repetitive that's why I consolidated them in one if, but it makes more sense this way to make changes more easy in case there's a difference in the future between these cluster types.

It's highly unlikely the test will reach this part of the code if a wrong `RANCHER_LOCAL_CLUSTER_TYPE` is set but I've added an else anyways as requested:

```
>           pytest.fail("Wrong RANCHER_LOCAL_CLUSTER_TYPE {}"
                        .format(RANCHER_LOCAL_CLUSTER_TYPE))
E           Failed: Wrong RANCHER_LOCAL_CLUSTER_TYPE GKEX

v3_api/test_create_ha.py:291: Failed
=========================== 1 failed in 0.18 seconds ===========================

Process finished with exit code 1
```